build/Branding.props

```diff
@@ -13,16 +13,16 @@
                                    '$(Rid)' == 'fedora.24-x64' OR
                                    '$(Rid)' == 'opensuse.42.1-x64' ">$(Rid)</ProductMonikerRid>
     <ProductMonikerRid Condition=" '$(ProductMonikerRid)' == '' ">$(OSName)-$(Architecture)</ProductMonikerRid>
-    
+
     <ArtifactNameSdk>dotnet-sdk</ArtifactNameSdk>
     <ArtifactNameSdkDebug>dotnet-sdk-debug</ArtifactNameSdkDebug>
     <ArtifactNameCombinedHostHostFxrFrameworkSdk>dotnet-dev</ArtifactNameCombinedHostHostFxrFrameworkSdk>
-    <ArtifactNameCombinedFrameworkSdk>dotnet-sharedframework-sdk</ArtifactNameCombinedFrameworkSdk>
+    <ArtifactNameCombinedFrameworkSdk>dotnet-runtime-sdk</ArtifactNameCombinedFrameworkSdk>
```

Comment:
I'm not sure I understand the point of this output.  Is it really used?  If yes, what for?  I think the name is super confusing now:  `dotnet-runtime-sdk`.  It only contains the `sdk`, `shared`, `additionaldeps`, and `store` folders, which isn't really usable by itself.  You need a host and hostfxr to run anything.

https://github.com/dotnet/cli/blob/6ecc1fc6c3f58bde065413cc869fb7f97d296143/build/package/Layout.targets#L15-L18

/cc @livarcocc 

Can we just delete this "artifact"?